assets/src/components/mapPage.tsx

```diff
@@ -116,27 +127,32 @@ const MapPage = (): ReactElement<HTMLDivElement> => {
 
   return (
     <div
-      className={`m-map-page ${mobileDisplayClass} ${mobileMenuClass}`}
+      className={`m-map-page ${mobileDisplayClass} ${mobileMenuClass} inherit-box border-box`}
       aria-label="Search Map Page"
     >
       <div
         className="m-map-page__input-and-results"
         aria-label="Map Search Panel"
-        {...(vpcEnabled ? { hidden: true } : {})}
+        hidden={vpcEnabled === true}
```

Comment:
**suggestion:** I think you can remove the `=== true` here because `vpcEnabled` was created from the `Boolean` constructor already.

Reply:
I know I _can_ but I really prefer the explicitness of how it reads. I'm happy to remove it, but I do think this is better for future readers of the code.

Reply:
I guess the way I think of it is that the name `vpcEnabled` already connotes that it's a boolean, although sadly JS / TS lacks an actual convention for naming boolean values like putting a `?` at the end in Elixir so it's not explicit as it could be. Since it's purely subjective I'm fine with leaving it the way you prefer.